guides/interactivity.md

```diff
@@ -0,0 +1,227 @@
+# Interactivity Guide
+---
```

Comment:
I think there should be one section that explains custom functions, time, audio, midi, and mouse inputs, and introduces arrays as a shorthand for custom functions. This should be in "Quick reference" as it is a core function of hydra. I say this because you need to understand the syntax of custom functions in order to work with mouse, interactivity. and audio. I usually explain it as something like..."anything that is a number can also be a number that changes over time. using variables and custom functions allows you to make sketches that change over time or in response to external inputs such as mouse, audio, midi, etc. "
I am not sure what the heading of this section should be though:
"Dynamic inputs"
"Interactivity"
"Variable inputs"